lab_1_classify_by_unigrams/main.py

```diff
@@ -34,7 +42,20 @@ def calculate_frequencies(tokens: list[str] | None) -> dict[str, float] | None:
 
     In case of corrupt input arguments, None is returned
     """
-
+    if isinstance(tokens, list):
+        freq = {}
+        for i in tokens:
+            if isinstance(i, str):
+                if i in freq:
+                    continue
+                average_freq = tokens.count(i) / len(tokens)
```

Comment:
you can move it inside the next if and set to `freq[i]` without creating `average_freq` variable

Reply:
done